TheSyndicate/Scenes.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TheSyndicate
+{
+    public class Scene
+    {
+        public string Id { get; set; }
+        public string Text { get; set; }
+        public string[] Options { get; set; }
+        public Dictionary<int, string> Destinations { get; set; }
+        public Scene ActualDestination { get; set; }
+        public bool Start { get; set; }
+
+
+        public Scene(string id, string text, string[] options, Dictionary<int, string> destinations, Scene actualDestination, bool start)
+        {
+            this.Id = id;
+            this.Text = text;
+            this.Options = options;
+            this.Destinations = destinations;
+            this.ActualDestination = actualDestination;
+            this.Start = start;
+        }
+
+        void RenderText()
```

Comment:
Should we clear the console before rendering the text?

Reply:
Yes, will make change